cfme/configure/configuration/system_schedules.py

```diff
@@ -44,13 +44,34 @@ class SambaProtocolEntities(View):
     samba_confirm_password = Input(id='log_verify')
 
 
+class AWSS3ProtocolEntities(View):
```

Comment:
Since these are defined as separate view classes and then nested, I think they should be imported, not re-defined. Any specific reason to re-define them here?